doc/tutorial/index.rst

```diff
@@ -59,6 +59,7 @@ Further reading
 .. toctree::
 
     loading_and_saving
+    prng
```

Comment:
Should be after sparse, in the advanced section above?

Reply:
I changed the minor issues.